src/components/molecules/NavBar/NavBar.tsx

```diff
@@ -356,7 +375,11 @@ export const NavBar: React.FC<NavBarPropsType> = ({
 
       {/* @debt Remove back button from Navbar */}
       {hasBackButton && venue?.parentId && parentVenue?.name && (
-        <div className="back-map-btn" onClick={backToParentVenue}>
+        <div className="back-map-btn"
+        role="button"
+        tabIndex={0}
+        onKeyPress={(e) => handleKeyPress(e, backToParentVenue)}
```

Comment:
We shouldn’t need to explicitly handle key presses here if we use the correct base types that naturally support these navigations. Eg. Button